src/main/entities/weapons/HeavySword.java

```diff
@@ -0,0 +1,35 @@
+package entities.weapons;
+
+import entities.InteractableProperties;
+import entities.Variable;
+
+public class HeavySword extends SwingableWeapon{
+    /**
+     * constructs a Sword object
+     * @param id the id of the heavy sword
+     */
+    public HeavySword(String id) {
+        super(id);
+        addHitProbability();
+        addWeight();
+
+    }
+    /**
+     * Adds property hit probability with the probability value
+     */
+    @Override
+    public void addHitProbability() {
+        super.addProperty(InteractableProperties.HIT_PROB.name(), new Variable(60));
+    }
+
+    /**
+     * Adds property weight with the weight/damage value that is randomized
+     */
+    @Override
+    public void addWeight() {
+        int weight = 30 + (int)(Math.random() * (60 + 1));
+        super.addProperty(InteractableProperties.WEIGHT.name(), new Variable(weight));
+    }
+
+    public HeavySword(){}
```

Comment:
Okay, I am noticing you have all these empty constructors. Was this intentional for all of them?

Reply:
Yeah it was. Since all similar classes had them, I assumed that these empty constructors were used somewhere. But if it's not useful we can deffo get rid of it

Reply:
Yea well I can merge your PR then, and we can take a look at that later. I don't think other classes need that either seems useless?